README.MD

```diff
@@ -1053,6 +1099,25 @@ U-Boot is quite clunky with the pathnames so try different paths like
 `/bbb_fs/boot/*`, `daft/bbb_fs/boot/*` or `/daft/bbb_fs/boot/*`. If it starts
 booting but gets stuck it's probably a problem with NFS or file permissions.
 
+If you get an error when you try to tftp the files, check that the kernel files `vmlinuz` and `dtbs` e.g. `vmlinuz-4.4.9-ti-r25` match the ones you have.
+
+
+### 6.2 DHCP problems
+If you are having problems with the DHCP server, you can try the following steps:
+1. Check that your dnsmasq service is on. In Fedora this is done by executing the following command `systemctl status dnsmasq`. If it is not, you need to start it with `systemctl start dnsmasq`. If it still isn't working, check that you have completed all the steps when configuring the [Dnsmasq](#232-dnsmasq).
```

Comment:
You can drop the 'Fedora', most of the modern distros have systemd and these commands would be the same with all of them.